server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java

```diff
@@ -354,4 +354,32 @@ public void removeGroupMappingShouldDelete() throws ErrorMappingException, Recip
         assertThat(virtualUserTable.getMappings(user, domain))
             .isEqualTo(MappingsImpl.empty());
     }
+
+    @Test
+    public void listSourcesShouldReturnWhenHasMapping() throws RecipientRewriteTableException {
+        String user = "test";
```

Comment:
user should be a constant, address too .

Having these variables makes test reading harder...